praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/repository/MeldezeitraumRepository.java

```diff
@@ -1,8 +1,13 @@
 package de.muenchen.oss.praktikumsplaner.repository;
 
 import de.muenchen.oss.praktikumsplaner.domain.Meldezeitraum;
+import java.time.LocalDate;
 import java.util.UUID;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 public interface MeldezeitraumRepository extends CrudRepository<Meldezeitraum, UUID> {
+    @Query("SELECT e FROM Meldezeitraum e WHERE :date >= e.startZeitpunkt AND :date <= e.endZeitpunkt")
+    Meldezeitraum findMeldezeitraumByDate(@Param("date") LocalDate date);
```

Comment:
a meldezeitraum has two dates so that the name should be more clear how the query interacts with these dates